### ISSUE TITLE
YoastSEO.js: import the screen-reader-text CSS class from yoast-components

### ISSUE THREAD

**Issue by afercia** [Yoast/javascript#252]
In the `yoastseo` package, the `_analysis.scss` file has its own `screen-reader-text` CSS class:

https://github.com/Yoast/javascript/blob/6631fd1e51d49a1f23fd2c3a72987801171168f0/packages/yoastseo/css/_analysis.scss#L56-L62

This ruleset is out of date. The `screen-reader-text` CSS class is subject to changes once in a while. Ideally we should have a centralized location as canonical source of this CSS rule and use it across all projects for easier maintenance. 

I'd like to propose to import if from yoast-components as done for MyYoast in https://github.com/Yoast/my-yoast/pull/2485
